packages/expo-face-detector/src/FaceDetector.ts

```diff
@@ -113,20 +113,20 @@ export type FaceFeatureBounds = {
 
 // @docsMissing
 export enum FaceDetectorMode {
-  fast = 'fast',
-  accurate = 'accurate',
+  fast = 1,
```

Comment:
these values are passed directly to MLKit and need to be numeric, otherwise we could remap them on the native side at some other point if wed like

Reply:
it feels like we increased the maintenance burden here when moving away from using constants directly off of the native module like in https://github.com/expo/expo/blame/58dbe04040a89df999765429a5f1456c3ee4b9ce/packages/expo-face-detector/src/FaceDetector.ts#L65-L69

if these were to change in the future it'd be tough for us to catch except through manual tests. perhaps it's worth reverting to the old approach and typing this differently? cc @Simek

Reply:
100% agree - it is better to have these values coming from the native module

Reply:
The reason behind constants refactor was purely because previous Constants types were set to `any`, which was the problem (I have trite about it a bit more in the PR description):
* https://github.com/expo/expo/pull/14179/files#diff-2ec81d0631a9f223b7545515416073d05615f6f1d78c8e55ab6348dfe343ea98L51

It was not the ideal solutions, but the one which works. It would be nice if we can solve this issue in other way, and ensure that the exported native values will have the correct type.